elm/ords/extraction/date.py

```diff
@@ -72,20 +72,22 @@ async def parse(self, doc):
 
 def _parse_date(json_list):
     """Parse all date elements."""
-    year = _parse_date_element(
+    years = _parse_date_element(
         json_list,
         key="year",
         max_len=4,
         min_val=2000,
         max_val=float("inf"),
     )
-    month = _parse_date_element(
+    months = _parse_date_element(
         json_list, key="month", max_len=2, min_val=1, max_val=12
     )
-    day = _parse_date_element(
+    days = _parse_date_element(
         json_list, key="day", max_len=2, min_val=1, max_val=31
     )
 
+    year, month, day = _generate_date(zip(years, months, days))
```

Comment:
Any reason not to just do something like this here?

```suggestion
    year, month, day = max(zip(years, months, days))
```